cmd/rhoas/main.go

```diff
@@ -57,24 +52,16 @@ func main() {
 	}
 
 	err = rootCmd.Execute()
-	if debug.Enabled() {
-		build.CheckForUpdate(context.Background(), logger, localizer)
-	}
 	if err == nil {
-		return
-	}
-
-	if e, ok := kas.GetAPIError(err); ok {
-		logger.Error("Error:", e.GetReason())
 		if debug.Enabled() {
-			errJSON, _ := json.Marshal(e)
-			_ = dump.JSON(cmdFactory.IOStreams.ErrOut, errJSON)
+			build.CheckForUpdate(context.Background(), logger, localizer)
 		}
-		os.Exit(1)
+		return
 	}
 
-	if err = cmdutil.CheckSurveyError(err); err != nil {
-		logger.Error("Error:", err)
+	if err != nil {
```

Comment:
This brings patter that our commands should no longer log message when error happens but return error that is exact message of what have happened.

Reply:
None of our commands log the error, they all already return the error and let the main program handle it.